snippets/snippets.json

```diff
@@ -19,8 +19,8 @@
         "scope": "povray",
         "body": [
             "light_source {",
-            "\t<${2:10}, ${3:10}, ${4:-10}> // <x, y, z>",
-            "\tcolor <${5:1.0}, ${6:1.0},  ${7:1.0}> // <red, green, blue>",
+            "\t<${1:x}, ${2:y}, ${3:z}>",
+            "\tcolor <${4:1.0}, ${5:1.0},  ${6:1.0}> // <red, green, blue>",

```

Comment:
`"\tcolor <${4:red}, ${5:green},  ${6:blue}>",`

I like what you did for the other places with RGB information: lines 468, 498, 509, …

Reply:
I couldn't decide between hinting to the user what the parameters are (e.g. `${1:red}`) or leaving the existing default values.

My goal was to
* get rid of the excessive comments left by the snippets
* fix errors in the parameter ordering

I left the light sources defaulting to RGB 1.0, 1.0, 1.0 since that is a sensible default for them; I often start off with pure white lights and usually leave them that way. Replacing them with placeholders seemed like it would create more work for someone using the snippet.

For other parameters, such as x, y and z there seemed no sensible default, so I replaced them with placeholders instead. Foe example, in line 22 which you've quoted above, <10, 10, -10> is extremely unlikely to be where I want to place my light source.